site/content/docs/contributing/1.0-roadmap.md

```diff
@@ -15,25 +15,41 @@ as a project.
 
 High level goals for kind include:
 
-- [ ] Reach [1.0] status / 
-stability by the end of Q1 2019 / the Kubernetes 1.14 release cycle
+## Beta Requirements
+
+To reach "beta" [grade][deprecation-policy] kind needs to at minimum:
+
+- [x] Improve documentation (Though this will eternally be "In Progress" !)
+  - [X] create a documentation site - [#268]
+  - [ ] expand examples of using kind
+  - [x] cover known issues, debugging, work-arounds, etc.
+- [x] Reliably [pass][kind-conformance-dashboard] the Kubernetes [conformance tests]
+ - [x] [Certify][certified] Conformance
 - [x] Support multi-node clusters - [#117]
-- [ ] Support offline / air-gapped clusters
-  - [ ] pre-loaded / offline CNI - [#200]
-- [ ] Support mounting host directories - [#62]
-- [ ] Improve Windows support
+- [x] Support offline / air-gapped clusters
+  - [x] pre-loaded / offline CNI - [#200]
+- [x] Support mounting host directories - [#62]
+- [x] Improve Windows support
   - [x] add Windows binaries to releases - [#155]
-  - [ ] improve instructions for KUBECONFIG in particular
-- [ ] Improve documentation - In Progress ...
-  - [ ] expand examples of using kind
-  - [ ] cover known issues, debugging, work-arounds, etc.
-  - [X] create a documentation site - [#268]
-- [ ] Improving logging and debuggability
-  - [ ] enable audit-logging
-- [ ] Support non-amd64 architectures (namely ARM) - [#166]
+  - [x] improve instructions for KUBECONFIG in particular
+- [ ] Support usage as a properly versioned, supported, and documented library 
+  - [ ] Improving logging and debuggability 
```

Comment:
Requesting a change here: this is too vague.

What specific improvements need to be made to logging?  What features need to be implemented to make kind appropriately debuggable?  

Or maybe looked at from another angle, what problems are really hard to troubleshoot right now, that we would want to be able to troubleshoot before reaching beta?

Reply:
Fair:
- it's wildly inconsistent currently
- it's difficult to consume as a library
- errors and other output generally lack enough context to determine what failed.

There are a number of specific fixes I'm looking at here, I'd like to keep this _somewhat_ vague as to how we solve them, but it should be possible to debug kind without needing to modify it or use some external debugging tools, currently that's not always the case due to the poor logging / errors.

Reply:
What about something like this? Trying to think from perspective of: if someone wanted to help you get to beta, what could they help out with
```suggestion
- [ ] should be possible to troubleshoot kind without needing to modify kind or use external debugging tools
  - [ ] consistent logging (what is logged, when should it be logged, what levels are used)
  - [ ] errors have appropriate context
```